vllm/attention/backends/rocm_flash_attn.py

```diff
@@ -434,7 +481,8 @@ def _sdpa_attention(
                 key[:, start:end, :],
                 value[:, start:end, :],
                 dropout_p=0.0,
-                is_causal=True,
+                is_causal=False,
```

Comment:
Could you explain why `is_causal=False`?

Reply:
attn_mask doesn't work with is_casual=True